crates/base/src/rt_worker/worker.rs

```diff
@@ -269,13 +270,19 @@ impl Worker {
                                 )
                                 .await;
 
-                            let found_unexpected_error = result.is_err()
-                                || matches!(
-                                    result.as_ref(),
-                                    Ok(WorkerEvents::UncaughtException(_))
-                                );
+                            let maybe_uncaught_exception_event = match result.as_ref() {
+                                Ok(WorkerEvents::UncaughtException(ev)) => Some(ev.clone()),
+                                Err(err) => Some(UncaughtExceptionEvent {
+                                    cpu_time_used: 0,
```

Comment:
I guess this is here as a placeholder value and gets replaced later?

Reply:
@laktek 
That arm is the result of the return type signature of `handle_creation`.
In fact, that arm is unreachable.

As you can see in `default_handler.rs`, nowhere in the body of the `handle_creation` does it return an Err.

Reply:
ok, gotcha